docker/api/network.py

```diff
@@ -47,7 +47,7 @@ def create_network(self, name, driver=None, options=None, ipam=None,
             driver (str): Name of the driver used to create the network
             options (dict): Driver options as a key-value dictionary
             ipam (dict): Optional custom IP scheme for the network.
-                Created with :py:meth:`~docker.utils.create_ipam_config`.
+                Created with :py:class:`~docker.types.IPAMConfig`.
```

Comment:
This won't actually link to anywhere unless they are represented in a docs file somewhere.